CONTRIBUTING.md

```diff
@@ -129,46 +129,97 @@ class Factory:
 
 ### Example of configuration
 
-Finally, you need to add the new AI in the `configuration.json` file.
+Finally, you will have a `configuration` similar to:
 
 ```json
 {
     "pluginPreferences":{
         "default": "ollama"
     },
-    "ai-models":{
-        "ollama":
-        {
-            "base_url": "http://localhost:11434/api",
-            "models": ["mistral"],
-            "defaultModel": "mistral",
-            "modelFiles": {
-                "generate": {
-                    "default": "default-mistral-modelfile_generate",
-                    "@ditrit/kubernator-plugin": "default-kubernetes-mistral-modelfile_generate",
-                    "@ditrit/githubator-plugin": "default-githubactions-mistral-modelfile_generate"
-                },
-                "message": {
-                    "default": "default-mistral-modelfile_message",
-                    "@ditrit/kubernator-plugin": "default-kubernetes-mistral-modelfile_message",
-                    "@ditrit/githubator-plugin": "default-githubactions-mistral-modelfile_message"
-                }
+    "ollama":
+    {
+        "base_url": "http://localhost:11434/api",
+        "models": ["mistral"],
+        "defaultModel": "mistral",
+        "modelFiles": {
+            "generate": {
+                "default": "default-mistral-modelfile_generate",
+                "@ditrit/kubernator-plugin": "default-kubernetes-mistral-modelfile_generate",
+                "@ditrit/githubator-plugin": "default-githubactions-mistral-modelfile_generate"
+            },
+            "message": {
+                "default": "default-mistral-modelfile_message",
+                "@ditrit/kubernator-plugin": "default-kubernetes-mistral-modelfile_message",
+                "@ditrit/githubator-plugin": "default-githubactions-mistral-modelfile_message"
             }
-        },
-        "MyAI": {
-            "base_url": "http://localhost:8080/api",
-            "otherSetting": "value"
-        }    
+        }
+    },
+    "MyAI": {
+        "base_url": "http://localhost:8080/api",
+        "otherSetting": "value"
     }
 }
 ```
 
+A configuration_description.json file is essential for integrating any AI system with Leto-Modelizer-Admin, as it defines the fields administrators must fill out to configure the AI properly.
+
+Here are the description of the configuration fields:
+
+- **`handler`**  
+  - **Purpose**: Identifies the AI system or plugin this configuration belongs to (e.g., "ollama").  
+  - **Use**: Associates the field with a specific handler or module.
+
+- **`key`**  
+  - **Purpose**: A unique identifier for the field used internally by the system.  
+  - **Use**: Represents the configuration parameter (e.g., `base_url`, `defaultModel`).
+
+- **`type`**  
+  - **Purpose**: Specifies the type of input expected in the UI.  
+  - **Values**:  
+    - `"text"`: Single-line text input.  
+    - `"checkbox"`: Toggleable option (true/false).  
```

Comment:
there is no "checkbox" replace it by select

Reply:
done